docker-tools/Dockerfile-from-conda

```diff
@@ -28,6 +28,6 @@ RUN python -c "import tornado; print('tornado version=' + tornado.version)"
 # Workaround, just calling `bokeh info` crashes
 RUN env BOKEH_RESOURCES=cdn bokeh info
 
-ENV BOKEH_EXAMPLE_DIR bokeh_examples
-COPY hello_world.py $BOKEH_EXAMPLE_DIR/hello_world.py
-COPY hello_nodejs.py $BOKEH_EXAMPLE_DIR/hello_nodejs.py
+#ENV BOKEH_EXAMPLE_DIR bokeh_examples
+#COPY docker_tools/hello_world.py $BOKEH_EXAMPLE_DIR/hello_world.py
+#COPY docker_tools/hello_nodejs.py $BOKEH_EXAMPLE_DIR/hello_nodejs.py
```

Comment:
should not leave in commented code

Reply:
Ah yeah, actually I should discuss this. Do we want these 'hello_world' scripts in the containers? They just plot something so you can test bokeh in the docker container. If so, should they maybe live in the bokeh examples or somewhere else maybe?

Reply:
I don't have a strong preference. They seem a little simplistic so I only wonder if it's possible to make something that is a nice intro example (so, fairly simple) but also somewhat compelling and interesting.

Reply:
I was trying to have an example that had the least chance of breaking between bokeh versions, as it would be nice to be able to generate containers for arbitrary bokeh versions with more-or-less the same scripts, as well as a quick way to test if bokeh 'works'.

Reply:
So the "iris" example in the examples dir has run unchanged for 5+ years. A lot of the bar examples there are unchanged for 2+ years. At this point any basic plot should safe at this point. Even "plot plus a wideget and CusomJS" should be fine now.

Reply:
We can also add whatever scripts get copied in to our CI testing to make sure they don't ever break

Reply:
I ended up just copying the app and models example folders, that'll be enough